### DISCUSSION TITLE
Does RACCOON require pre-defined cracks?

### DISCUSSION THREAD

**Discussion by NeedOfHelp** [idaholab/moose#23883] [Q&A Modules: Phase field] [answered]
Hi,

As the title says, when modelling using RACCOON, does the model require a pre-defined crack?
If not, is there any example code i can look at where it shows a new crack forming? 

Thanks and cheers.

**Comment by GiudGiud** [comment]
@hugary1995

**Reply by GiudGiud** [reply]
(if there are others knowledgeable please let me know so I can spread the tagging)

**Reply by hugary1995** [reply, marked as answer]
You could also tag @jiangwen84 for phase field fracture (RACCOON) questions. I'm a little behind on other stuff so I probably won't monitor this discussion page frequently enough.

@NeedOfHelp pre-defined crack in which representation? Regularized (phase-field) crack or sharp crack (mesh discontinuity)?

For a regularized crack, there are a few initial condition objects in RACCOON you can play with. More generally speaking, you can use any MOOSE initial condition object to set the initial phase field.

For a sharp crack, you need to prepare a meshed-in crack.

**Reply by NeedOfHelp** [reply]
Thanks for your reply. 
I was thinking more in phase-field modelling of cracks. I have seen your tutorial codes on fracture and all of them seems to have a pre-defined crack, as far as I have seen.
So I started wondering how the simulations would go if the crack was defined in the middle, the sides, two cracks at the same time and also what if there were no initial crack at all?

I'm a new user to RACCOON and MOOSE and I'm trying to learn my way up, so I am kinda clueless on what types of modules/packages that exists.

Cheers.
